sdk/search/azure-search-documents/azure/search/documents/__init__.py

```diff
@@ -43,9 +43,9 @@
     edm,
 )
 from ._service._generated.models import (
-    Analyzer,
     AnalyzeRequest,
```

Comment:
Hard to tell from the diff, but these *Request and *Result classes aren't actually exported, are they? They should most often (always, IIRC) be wrapped/transformed before returning.

Reply:
Same for `DataSourceCredentials` I see below.